### PR TITLE
Change groupId to com.azure and artifactId to azure-spring-cloud-starter-xxx

### PR DESCRIPTION
**Pull request by ZhuXiaoBing-cn**
update #13523

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-cosmosdb/README.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/CONTRIBUTING.md) to build from source or contribute.
+
+<!-- LINKS -->
+[docs]: https://docs.microsoft.com/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db
+[refdocs]: https://azure.github.io/azure-sdk-for-java/spring.html#azure-cosmosdb-spring-boot-starter
+[package]: https://mvnrepository.com/artifact/com.microsoft.azure/azure-cosmosdb-spring-boot-starter
+[sample]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-cosmosdb
```
Comment (moarychan) [inline sdk/spring/azure-spring-boot-starter-cosmosdb/README.md:249]: should  sync code from azure master first

File: sdk/spring/azure-spring-cloud-autoconfigure/pom.xml [hunk 2]
```diff
@@ -210,10 +210,10 @@
...
-                <include>com.microsoft.azure:spring-integration-eventhubs:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-integration-eventhubs;current} -->
-                <include>com.microsoft.azure:spring-integration-servicebus:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-integration-servicebus;current} -->
+                <include>com.azure:azure-spring-cloud-starter-context:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-cloud-starter-context;current} -->
+                <include>com.azure:azure-spring-cloud-starter-telemetry:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-cloud-starter-telemetry;current} -->
+                <include>com.azure:azure-spring-integration-eventhubs:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-integration-eventhubs;current} -->
+                <include>com.azure:azure-spring-integration-servicebus:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-integration-servicebus;current} -->
```
Comment (moarychan) [inline sdk/spring/azure-spring-cloud-autoconfigure/pom.xml:216]: we can remove all the dependencies which the group id starts with `com.azure:xxx` in `bannedDependencies` part.

**Review by moarychan — CHANGES_REQUESTED** [review]
please comfirm with @wujack778  to avoid repeat modification.

**Comment by moarychan** [comment]
this file https://github.com/ZhuXiaoBing-cn/azure-sdk-for-java/blob/work/sdk/spring/ci.yml should be updated together.

**Comment by ZhuXiaoBing-cn** [comment]
I am sorry that there are some problems in my repository and there are redundant submissions. I need to close this PR now.

**Comment by chenrujun** [comment]
Hi, @ZhuXiaoBing-cn , you can use `git push -f` to avoid creating `closed` PR.

**Pull request closed without merging**